um/.htaccess

```diff
@@ -0,0 +1,8 @@
+Header set Access-Control-Allow-Origin *
+Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^(.*)/cbcm/eu-cm-ontology(.*)$ https://maastrichtu-ids.github.io/cbcm-ontology$2 [R=302,L]
+RewriteRule ^(.*)/ids/shapes(.*)$ https://maastrichtu-ids.github.io/shapes-of-you/ [R=302,L]
+RewriteRule ^(.*)/ids/projects(.*)$ https://maastrichtu-ids.github.io/projects/ [R=302,L]
```

Comment:
Did you intend to use those wildcard prefixes?  There are edge cases for some urls that could lead to odd behavior.  Like /um/ids/projects/ids/shapes would match the shapes rule first.